test/unit/mdc-checkbox/mdc-checkbox.test.js

```diff
@@ -263,3 +257,16 @@ test('#adapter.setNativeControlDisabled returns false when checkbox is not disab
   component.getDefaultFoundation().adapter_.setNativeControlDisabled(false);
   assert.isFalse(cb.disabled);
 });
+
+
+test('"checked" property change hook works correctly', () => {
```

Comment:
These tests aren't actually testing the property change logic whatsoever. These would pass even if the hooks were never installed.

I'm working on fixing this.